README.md

```diff
@@ -187,18 +205,72 @@ docker exec -ti eventyay-talk bash
 >
 > ```bash
 > # Initialize talk system with a superuser account
-> cd ~
```

Comment:
That should not be deleted, since you will be dropped into a different directory